src/cross_section.F90

```diff
@@ -248,6 +267,12 @@ subroutine calculate_nuclide_xs(i_nuclide, i_sab, E, i_mat, i_nuc_mat, u)
       micro_xs(i_nuclide) % kappa_fission = &
            nuc % reactions(nuc % index_fission(1)) % Q_value * &
            micro_xs(i_nuclide) % fission
+
+      ! Calculate microscopic nuclide delayed nu-fission cross section
+      micro_xs(i_nuclide) % delayed_nu_fission = (ONE - f) * &
+           nuc % delayed_nu_fission(i_grid) + f * &
+           nuc % delayed_nu_fission(i_grid+1)
```

Comment:
Removing the macroscopic `delayed_nu_fission` xs calculation in `calculate_xs` is not a big deal. The macro xs is only needed when performing a track length or collision tally on the total nuclides, and the tally routine can just be modified to loop over all nuclides in a material and tally the contributions from each nuclide. Removing the microscopic `delayed_nu_fission` xs from the `calculate_nuclide_xs` subroutine would require a new subroutine to calculate the nuclide delayed-nu-fission microscopic xs. While this is not difficult to implement, it results in the cross sections not being completely in sync with each other. When a delayed-nu-fission score is needed, the nuclide delayed-nu-fission microscopic xs would need to be recomputed, which could be confusing to require this dependency for only one tally score.

Would it be okay to keep in the delayed-nu-fission micro xs calculation, but remove the delayed-nu-fission macro xs calculation? Is there any profiling you would recommend doing to investigate the performance drop of calculating the `delayed-nu-fission` microscopic xs in the `calculate_xs` subroutine?